Common/cpp/Tools/RuntimeDecorator.cpp

```diff
@@ -57,25 +70,7 @@ void RuntimeDecorator::decorateRuntime(
           evalWithSourceUrl));
 #endif // DEBUG
 
-  auto callback = [](jsi::Runtime &rt,
-                     const jsi::Value &thisValue,
-                     const jsi::Value *args,
-                     size_t count) -> jsi::Value {
-    const jsi::Value *value = &args[0];
-    if (value->isString()) {
-      Logger::log(value->getString(rt).utf8(rt).c_str());
-    } else if (value->isNumber()) {
-      Logger::log(value->getNumber());
-    } else if (value->isUndefined()) {
-      Logger::log("undefined");
-    } else {
-      Logger::log("unsupported value type");
-    }
-    return jsi::Value::undefined();
-  };
-  jsi::Value log = jsi::Function::createFromHostFunction(
-      rt, jsi::PropNameID::forAscii(rt, "_log"), 1, callback);
-  rt.global().setProperty(rt, "_log", log);
+  jsi_utils::installJsiFunction(rt, "_log", logValue);
 
   auto chronoNow = [](jsi::Runtime &rt,
```

Comment:
For `chronoNow`, we also add it as a property on the `performance` object, with a different name.
At best, we could use `createHostFunction` to wrap a parameterless lambda here and assign that to `chronoNow`, but I'm not sure that would be any more readable.

For `evalWithSourceUrl`, that function checks the number of arguments it was passed from JS, and the helpers don't allow for that.
There's also little reason to do that in my opinion, since you can't exactly do much better than pass a pointer and count of elements when passing a variable number of arguments.